source/ulid.ts

```diff
@@ -265,3 +267,38 @@ export function ulid(seedTime?: number, prng?: PRNG): ULID {
     const seed = isNaN(seedTime) ? Date.now() : seedTime;
     return encodeTime(seed, TIME_LEN) + encodeRandom(RANDOM_LEN, currentPRNG);
 }
+
+export function ULIDtoUUID(ulid: string): UUID {
```

Comment:
Would you mind renaming this to `ulidToUUID`? I'd prefer to keep simple functions lower-case for the first portion.

Reply:
Done!